diff --git a/node_modules/mutik/dist/index.d.ts b/node_modules/mutik/dist/index.d.ts
index ad1557b..ed309c7 100644
--- a/node_modules/mutik/dist/index.d.ts
+++ b/node_modules/mutik/dist/index.d.ts
@@ -15,5 +15,5 @@ export declare function Provider<S>({ children, store, }: {
     children: React.ReactNode;
     store: Store<S>;
 }): React.FunctionComponentElement<React.ProviderProps<Store<unknown>>>;
-export declare function useSelector<S, V>(selector: (s: S) => V): any;
+export declare function useSelector<S, V>(selector: (s: S) => V): V;
 export {};
